Package.swift

```diff
@@ -15,6 +15,10 @@ let package = Package(
             path: "SwiftMessages",
             exclude: [
                 "Info.plist",
+            ],
+            resources: [
```

Comment:
Is it necessary to add this item? In my test scenarios it was not.

Reply:
Ah, maybe not, tried this before I found out that the bundle was wrong.

Reply:
Removed and works!